c/tskit/tables.h

```diff
@@ -4004,6 +4004,10 @@ life-cycle.
 bool tsk_table_collection_has_index(
     const tsk_table_collection_t *self, tsk_flags_t options);
 
+/* TODO not clear this should have the "options" flags. */
+bool tsk_table_collection_has_reference_sequence(
+    const tsk_table_collection_t *self, tsk_flags_t options);
```

Comment:
I can only imagine this having options if we use it to check fetch status, but I think that would be clearer as an additional function.